python/tests/dispersive_eigenmode.py

```diff
@@ -0,0 +1,147 @@
+
+# dispersive_eigenmode.py - Tests the meep eigenmode features (eigenmode source,
+# eigenmode decomposition, and get_eigenmode) with dispersive materials.
+# TODO: 
+#  * check materials with off diagonal components
+#  * check magnetic profiles
+#  * once imaginary component is supported, check that
+
+from __future__ import division
+
+import unittest
+import meep as mp
+import numpy as np
+from meep import mpb
+import h5py
+class TestDispersiveEigenmode(unittest.TestCase):
+    # ----------------------------------------- #
+    # ----------- Helper Functions ------------ #
+    # ----------------------------------------- #
+    # Directly cals the C++ chi1 routine
+    def call_chi1(self,material,omega):
```

Comment:
would be nice to have a documented API for this.    The natural thing would be an `epsilon(omega)` and `mu(omega)` method of material, but are these names already taken by fields?

Reply:
Oh, nevermind, we already have these.